### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
Like #30 but "better" (quotation marks used because that's subjective). Reasons I think it is better:

- Only builds on Linux.
- Only builds on JDK 11
- Simpler/smaller file?

### CODE REVIEW

**Comment by katekerllenevich** [comment]
Build on java 11 + 8 or just 11.

**Comment by DragonEggBedrockBreaking** [comment]
> Build on java 11 + 8 or just 11.

Done.

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 8
+        java-version: 11
```
Comment (ImperatorStorm) [inline .github/workflows/gradle.yml:14]: can i get a rationale behind using java 11 and not 8?

Comment (katekerllenevich) [reply .github/workflows/gradle.yml:14]: java 8 is deprecated and is end of life. java 11 provides many qol features/changes and a significant performance boost. everything will remain backwards compatible.

Comment (DragonEggBedrockBreaking) [reply .github/workflows/gradle.yml:14]: ask @notOM3GA (who suggested it)
i didnt know what to use, assumed they knew better than me, and changed it
i know that you are smart, and i have no justification, so should i change it back, or what version do you suggest

Comment (DragonEggBedrockBreaking) [reply .github/workflows/gradle.yml:14]: > java 8 is deprecated and is end of life. java 11 provides many qol features/changes and a significant performance boost. everything will remain backwards compatible.

Ah, OK, so here is the justification (didnt see this)

Comment (DragonEggBedrockBreaking) [reply .github/workflows/gradle.yml:14]: @ImperatorStorm So what should I do?

Comment (katekerllenevich) [reply .github/workflows/gradle.yml:14]: remain on java 11. they were just asking why 11 was being used.

Comment (DragonEggBedrockBreaking) [reply .github/workflows/gradle.yml:14]: ok

Comment (ImperatorStorm) [reply .github/workflows/gradle.yml:14]: I'd stay on java 8, as most people who would use this mod would most likely use java 8

Comment (katekerllenevich) [reply .github/workflows/gradle.yml:14]: Java has great backwards compatibility. It will still work on Java 8.

Comment (ImperatorStorm) [reply .github/workflows/gradle.yml:14]: i'll be damned, it works.

Comment (DragonEggBedrockBreaking) [reply .github/workflows/gradle.yml:14]: > i'll be damned, it works.

OK, cool. Btw, if possible, could you chat further somewhere else, as this is kinda spamming my inbox :-)

**Comment by DragonEggBedrockBreaking** [comment]
Updated loom version, because otherwise some mods don't build. I know little about why, how, etc. but I know that updating loom to `0.6-SNAPSHOT` fixes it, and I also know that it is the recommended version to use, so I am including it in this PR. Feel free to cherry-pick only the GitHub Actions commits from my fork, if you do not want this change.

**Comment by DragonEggBedrockBreaking** [comment]
Added a fix to #28 so that this commit has all the features of #30

**Comment by DragonEggBedrockBreaking** [comment]
This commit history is very messy, I will re-open the PR.

**Pull request closed without merging**